packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -439,6 +462,9 @@ export function createEditPageNode({
                                         />
                                     )}
                                 </Field>
+
+                                <CustomDivider />
```

Comment:
This Divider will be rendered even if there are no additionalFormFields (which come from the app). The divider should be rendered by the app I suppose

Reply:
So should I only render it if additional form fields are present or how should I proceed with this?

Reply:
Alright thanks 👍🏻 will do